static/app/components/sidebar/sidebarAccordion.tsx

```diff
@@ -78,6 +81,37 @@ function SidebarAccordion({children, ...itemProps}: SidebarAccordionProps) {
 
 export {SidebarAccordion};
 
+function recursivelyFindChildren(
```

Comment:
I was thinking was that if we had a slot like API here, we would be able to avoid searching the tree for children (as they would self register with the parent). It wouldn't be slots in the traditional sense that they'd have to be unique, but they could just register with the parent component